_posts/en/newsletters/2024-12-13-newsletter.md

```diff
@@ -0,0 +1,308 @@
+  [exogenous][topic fee sourcing] fee sourcing, e.g. all states paying
+  the same commitment transaction fee, which depends on robust package
+  relay for [CPFP][topic cpfp] fee bumping in the Bitcoin P2P protocol
+  and channel commitment transaction upgrades for LN.
+
+- **Insights into channel depletion:** René Pickhardt [posted][pickhardt
+  deplete] to Delving Bitcoin and [participated][FIXME], along with
+  Christian Decker, in an Optech Deep Dive about his research into the
+  mathematical foundations of payment channel networks (namely LN).  A
+  particular focus of his Delving Bitcoin post was a discovery that some
+  channels in circular paths on the network will eventually become
+  depleted if the path is used enough.  When a channel is effectively
+  fully depleted, meaning it cannot forward additional payments in the
+  depleted direction, the circular path (cycle) is broken.  As each
+  cycle in the network graph is successively broken, the network
+  converges on a residual graph that has no cycles (a spanning tree).
+  This replicates a [prior result][guidi spanning] from researcher
+  Gregorio Guidi, although Pickhardt arrived at it from a different
+  approach, and also appears to be confirmed in unpublished research by
+  Anastasios Sidiropoulos.
+
+  ![Example of cycles, depletion, and residual spanning tree](/img/posts/2024-12-depletion.png)
+
+  Perhaps the most notable insight provided by this result is that
+  widespread channel depletion occurs even in a circular economy where
+  there are no source nodes (i.e., net spenders) and sink nodes (i.e.,
+  net receivers).  If LN was used for every
+  payment---customer-to-business, business-to-business, and
+  business-to-worker---it would still converge on a spanning tree.
+
+  It's not clear whether nodes would want their channels to be part of
+  the residual spanning tree or not.  One one hand, that tree represents
+  the last part of the network that would still be capable of forwarding
+  payments---it's equivalent to a hub-and-spoke topology---so it may be
+  possible to charge high forwarding fees across the residual channels.
+  On the other hand, the residual channels are what's left after all
+  other channels have already collected fees to depletion.
+
+  Although a channel with higher forwarding fees is less likely to
+  become depleted (all other things being equal), the properties of
+  other channels in the same cycles strongly influence the likelihood of
+  depletion, making it challenging for a node operator to attempt to use
+  control over their own forwarding fees alone to prevent depletion.
+
+  Higher capacity channels are also less likely to become depleted than
+  lower capacity channels.  That seems obvious, but carefully
+  considering the reason for it yields a surprising insight about k>2
+  multiparty offchain protocols.  A higher capacity channel supports a
+  greater number of wealth distributions between its participants, so
```

Comment:
```suggestion
  greater variety of wealth distributions between its participants, so
```
?

Reply:
I prefer number.  "Variety" implies to me non-equivalence between distributions, e.g. {{Larry: flowers, Dave: apples}, {Larry: horses, Dave: sand}}.  What we're trying to imply here is name of states where the same total amount of wealth can be divided, all of which total to 100%, e.g. {{Larry: 1, Dave: 2}, {Larry: 2, Dave: 1}}.  "Number" feels more natural to me for that.